package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npx @vercel/ncc@0.34.0 build index.js --license LICENSE"
```

Comment:
Can you elaborate on what this does and why you think adding this is necessary ?

Reply:
No problem, very sensible. It was copied from the upstream where it is used as a way to generate the build, https://github.com/peter-murray/issue-body-parser-action/blob/main/package.json#L8